### DISCUSSION TITLE
Sync Interval per Calendar (and not only a single general setting)

### DISCUSSION THREAD

**Discussion by thn80** [bitfireAT/icsx5#364] [Ideas]
It would be nice if ICSx5 would support a sync interval setting per calendar and not only the single global sync interval setting for all calendars.

I have two calendars on my smartphone which are handled by ICSx5, for one of the calendars it is sufficient to sync it one a day. However, for the other calendar a higher sync interval (e.g. 15 minutes or 1 hour) is required. To keep the load low on the server which is only required to be synched once a day, it would be nice to have the option to configure the sync interval per calendar.

It would also be nice to have the feature that some calendars can sync per configured interval, but other calendars can be synched only manualls. Meaning, to have a checkbox to enable/disable periodically synching for specific calenders.

**Comment by sunkup** [comment]
Thanks for your suggestion. This sounds very specific and since server load in the case of calendar syncing is often quite minimal I doubt many people need to optimize their calendar syncs that way. Unless many people need the feature, I don't think we will add this to ICSx5. We plan on integrating ICSx5 into DAVx5 and maybe it will come as functionality at some point then.
